packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -6,8 +6,20 @@ import {
   h,
   Event,
   EventEmitter,
+  forceUpdate,
 } from '@stencil/core';
+import i18next from 'i18next';
+// import { i18next } from '../../../../core/src/i18n/i18n-setup';
 import { consoleDevError } from '../../utils/utils';
+import { Build } from '@stencil/core';
+
+/** The .t() function will return the key instead of the value
+ */
+if (Build.isTesting) {
+  i18next.init({ lng: 'cimode' });
+}
```

Comment:
I don't like having this in the component's definition file. I originally tried having the `cimode` get set in the test file, but I think the instance of `i18next` in the compiled bundle is different than the one that's getting imported in the test file, so I wasn't initializing the right thing. :disappointed: